wordpress-mobile/WordPress-FluxC-Android
# Issue-3605. Example of customer creation

Part number 9 of [#3605](https://github.com/woocommerce/woocommerce-android/issues/3605), contains an example of the usage of FluxC functionality of customer creation

**Testing:**
1. Open the example
2. Sign in
3. Click "WOO" button
4. Scroll down and click "CUSTOMERS" button
5. Select site
6. Click "CREATE CUSTOMER"
7. Type data you need
8. Click "CREATE" button
9. Check the logs

<img src="https://user-images.githubusercontent.com/4923871/109984544-ef261380-7d14-11eb-9deb-5effabd57e03.png" width="300"/>
